eole/transforms/tokenize.py

```diff
@@ -563,6 +574,10 @@ def _detokenize(self, tokens, side="src", is_train=False):
             )
         else:
             detokenized = tokenizer.detokenize(tokens)
+            # let's match one or more repetitions of the <0x..> pattern
+            detokenized = re.sub(
+                r"(?:<0x[0-9A-Fa-f]{2}>\s*)+", hex_to_char, detokenized
+            )
```

Comment:
Not sure we want to support the full complexity of the HF implementation, but might be good to at least have some try/except at some level to prevent failing the full prediction for some localized issue(s).